readthedocs/proxito/tests/test_hosting.py

```diff
@@ -845,6 +849,90 @@ def test_number_of_queries_url_translations(self):
             )
         assert r.status_code == 200
 
+    @mock.patch("readthedocs.filetreediff.get_manifest")
+    def test_file_tree_diff(self, get_manifest):
+        get(
+            Feature,
+            projects=[self.project],
+            feature_id=Feature.GENERATE_MANIFEST_FOR_FILE_TREE_DIFF,
+        )
+        pr_version = get(
+            Version,
+            project=self.project,
+            slug="123",
+            active=True,
+            built=True,
+            privacy_level=PUBLIC,
+            type=EXTERNAL,
+        )
+        pr_build = get(
+            Build,
+            project=self.project,
+            version=pr_version,
+            commit="a1b2c3",
+            state=BUILD_STATE_FINISHED,
+            success=True,
+        )
+        get_manifest.side_effect = [
```

Comment:
Do we want side_effect here instead of a return value? I haven't used side_effect like this before.

Reply:
side_effect allows you to return multiple values (one for each call), while return_value allows only returning the same value.